templates/index.html

```diff
@@ -669,7 +669,7 @@ <h2 class="p-link--external p-muted-heading">
         {% if insights_feed %}
           {% for item in insights_feed %}
             <article class="col-4 p-divider__block">
-              <h3 class="p-heading--four"><a href="{{ item.link }}" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'blog', 'eventAction' : 'maas.io homepage news link', 'eventLabel' : '{{ item.title|escape }}', 'eventValue' : '{{ item.link }}' });">{{ item.title }}</a></h3>
+              <h3 class="p-heading--four"><a href="{{ item.link.replace('admin.insights', 'blog') }}" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'blog', 'eventAction' : 'maas.io homepage news link', 'eventLabel' : '{{ item.title|escape }}', 'eventValue' : '{{ item.link }}' });">{{ item.title }}</a></h3>
```

Comment:
it's now hosted on ubuntu.com/blog no on blog.ubuntu.com anymore :)

Reply:
While I knew `blog.ubuntu.com/<date>/<slug>` was redirecting to `ubuntu.com/blog/<slug>`, wasn't sure about `ubuntu.com/blog/<date>/<slug>` doing the same. Works fine actually. Changed.